uwblueprint/focus-on-nature
# feat: camp overview information

## Notion ticket link

[Camp Overview: Camp level information view only](https://www.notion.so/uwblueprintexecs/Camp-Overview-Camp-level-information-view-only-343f864f624945edacc3d6fe599c0ebf)



## Implementation description
* Created a camp overview page that just includes the top-level camp information according to the Figma designs
* Added an APIClient to call the BE APIs to get the camp information
* There are some aspects of this overview that are out of scope of this ticket and some things that are blocked too (check Additional Notes & Clarifications in the Notion ticket to see what's included and not included)
* Currently a camp is just hard coded for now
* Some "TODOs" have been called out as comments in the code (left those there as placeholders for future updates that were deemed out-of-scope for this ticket)



## Steps to test
1. Build containers and open the page at http://localhost:3000/admin/camp/62c098e7b4a7a433a7622ff4 on a browser
2. Check that this matches the Figma designs and that the right information is pulled from the DB



## What should reviewers focus on?
* Code quality and placement


## Checklist
